File: medium/pro-roadrunner/Roadrunner.rb
distanceApart = 50
distanceSafety = gets.chomp.to_i
speedRoadrunner = gets.chomp.to_i
speedCoyote = gets.chomp.to_i
        
timeToSafety = distanceSafety.to_f / speedRoadrunner
        
isCoyoteCaughtRoadrunner = speedCoyote > (distanceApart + distanceSafety) / timeToSafety
        
puts isCoyoteCaughtRoadrunner ? "Yum": "Meep Meep"

File: medium/pro-roadrunner/roadrunner.c
#include <stdio.h>

int main() {
    int const distanceApart = 50;
    int distanceSafety, speedRoadrunner, speedCoyote;
    scanf("%d\n%d\n%d", &distanceSafety, &speedRoadrunner, &speedCoyote);
    
    float timeToSafety = (float) distanceSafety / speedRoadrunner;
    int isCoyoteCaughtRoadrunner = speedCoyote > (distanceApart + distanceSafety) / timeToSafety;
    puts(isCoyoteCaughtRoadrunner ? "Yum" : "Meep Meep");
    
    return 0;
}
